routes.jsx

```diff
@@ -15,21 +15,38 @@ import Navbar from './components/navbar/navbar.jsx';
 import Footer from './components/footer/footer.jsx';
 
 const App = React.createClass({
+  getInitialState() {
+    return {
+      suppressSplashScreen: false
+    };
+  },
   componentDidMount() {
-    setTimeout(function() {
-      this.splash.classList.add(`dismissed`);
-      document.querySelector(`#app`).classList.add(`splash-dismissed`);
-    }, 3000);
+    const suppressSplashScreenKey = `suppressSplashScreen`;
+
+    this.setState({
```

Comment:
I'm getting server side error `ReferenceError: localStorage is not defined` if I try to access `localStorage` before `componentDidMount`. Should I shim it as what you did in `app-user.js` ?